src/TRestGeant4NeutronTaggingProcess.cxx

```diff
@@ -244,8 +244,7 @@ TRestEvent* TRestGeant4NeutronTaggingProcess::ProcessEvent(TRestEvent* evInput)
     // veto groups
     for (const auto& pair : fVetoGroupVolumeNames) {

```

Comment:
Still not structure bindings (C++17)?
const auto& [param1, param2] : fVetoGroupVolumeNames is more readable